src/adapters/head_utils.py

```diff
@@ -369,6 +369,35 @@
         },
         "layers": ["lm_head"],
     },
+    # PLBART
+    "PLBartForSequenceClassification": {
+        "config": {
+            "head_type": "classification",
+            "layers": 2,
+            "activation_function": "tanh",
+        },
+        "layers": [
+            None,
+            "classification_head.dense",
+            None,
+            None,
+            "classification_head.out_proj",
+        ],
+    },
+    "PLBartForQuestionAnswering": {
```

Comment:
This class does not seem to be part of Transformers? If it isn't we can remove this

Reply:
Yes, I believe it is a part of the Transformers. You can refer to this [link](https://github.com/huggingface/transformers/blob/1082361a1978d30db5c3932d1ee08914d74d9697/src/transformers/models/plbart/modeling_plbart.py#L1433). Also, IIRC, one of the tests failed because it was missing. Let me know if there is something to be modified.

Reply:
PLBartForQuestionAnswering does not seem to be part of Transformers though or am I missing something?

Reply:
Yes, you are correct. It's removed now. My bad, I thought you were referring to PLBartForSequenceClassification.